modules/drivers/bigquery-cloud-sdk/test/metabase/driver/bigquery_cloud_sdk_test.clj

```diff
@@ -309,37 +310,83 @@
             (doall (map drop-table-if-exists! (keys table-name->is-filter-required?)))
             nil)))))))
 
-(deftest sync-pseudocolumn-for-ingestion-time-partitioned-table-test
-  (testing "for ingestion time partitioned tables, we should sync the pseudocolumn _PARTITIONTIME AND _PARTITIONDATE"
-    (mt/test-driver :bigquery-cloud-sdk
+(deftest full-sync-partitioned-table-test
```

Comment:
I incorporated the existing test `sync-pseudocolumn-for-ingestion-time-partitioned-table-test` into this one as it could reuse some of the setup.

Creating a table in bigquery is a slow process so I don't want to do that too many times.

Reply:
Very nice test.